GeneralsMD/Code/GameEngine/Include/Common/AcademyStats.h

```diff
@@ -68,7 +68,7 @@ struct AcademyAdviceInfo
 	UnsignedInt numTips;
 };
 
-enum AcademyClassificationType
+enum AcademyClassificationType CPP_11(: int)
```

Comment:
Why are you specifying the underlying type for enums that are fully defined? Specifying the type is optional?

Reply:
When you use type on forward declarations of enum, you then need to use it also for definitions. Inconsistency here can lead to compiler errors as well.

Reply:
Given the scale of the issue, I have not explicitly checked existence of forward declarations of each enum, but simply changed them all (by script). I think changing all is better anyway, because in other case you would have some enums you can forward declare and some which you cant...

Reply:
I agree with consistency.